libs/ui/src/components/atoms/button/button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ButtonBase } from './buttonStyles';
+import classNames from 'classnames';
+
+type ButtonProps = {
+  children: React.ReactNode;
+  secondary: boolean;
+  sm: boolean;
+  lg: boolean;
+  tertiary: boolean;
+  disabled: boolean;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+};
```

Comment:
Should some of these be combined? We can't have a lg and small button or a secondary and tertiary button correct?

Reply:
Right. This follows the spec though in keeping the props minimalistic booleans. 

I think we could trust devs to not set two styles or colors at once. Also worth mentioning that we could possibly double set without this pattern as well.